README.md

```diff
@@ -1,113 +1,47 @@
 What This Is
 ============
+Long ago QUIC was created. For the past years Akamai and Microsoft have
+collaborated on maintaining QUIC support via the BoringSSL APIs. However,
+more and more features and needed improvements lingered for years.
```

Comment:
Perhaps "For the past couple of years, the OpenSSL project has been focused on their own implementation of QUIC, to the detriment of other features and improvements." ?  Or some other change, but it's not clear who the last sentence applies to.

Reply:
I've modified this sentence, hope the new one works better.